notebooks/15_neural_networks/metadata.yml

```diff
@@ -40,3 +40,27 @@ authors:
           link: https://github.com/nimajam41
         - icon: fas fa-envelope
           link: mailto:nimxj4141@gmail.com
+
```

Comment:
1. Please add the table of contents
2. Complete Example Codes
3. Please Add Conclusion
4. Make your references more specific and clickable